### DISCUSSION TITLE
Image.Identify disposal

### DISCUSSION THREAD

**Discussion by JasonAtATL** [SixLabors/ImageSharp#1938] [Show and tell] [answered]
Hello,

I wasn't sure if this should be submitted as an issue or a discussion, so I'm starting with the later.

We recently started using SixLabors.ImageSharp in our project, and there is some confusion about how Image.Identity handles IDisposable. The documentation indicates a using statement should be used when calling Image.Identify, but it doesn't appear IImageInfo implements IDisposable. Is this a typo?

https://docs.sixlabors.com/articles/imagesharp/imageformats.html#metadata-only-decoding

What the correct way to call Image.Identity to ensure resources are property disposed?

Thanks!

**Comment by JimBobSquarePants** [comment, marked as answer]
Hi @JasonAtATL 

Yep, that's a mistake in the documentation, well spotted! `IImageInfo` implementations don't require disposal.

I've created a PR in the docs to change this.
https://github.com/SixLabors/docs/pull/38

**Comment by JasonAtATL** [comment]
Thanks for the confirmation, @JimBobSquarePants!
